Assignment2/Cristinalonso29-CristinaAlonsoDelgado.ttl

```diff
@@ -0,0 +1,13 @@
+@base <https://www.oeg-upm.net/Organization#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>.
+@prefix ns: <http://www.ontologies.org/ontologies/Class#>.
+
+:Class01 :includes :Sensor029; :includes :Computer101.
+
+:Sensor029 :hasMeasurement :Measurement8401.
+
+:Measurement8401 :hasTemperature 29^^xmls:integer; :atTime 				2010-06-12T12:00:12^^xmls:dateTime.

```

Comment:
xmls prefix is not declared anywhere. It is wrong. This is coming from some files accepted last year.
Furthermore, this Turtle cannot be parsed. Do it before submitting.